test/graphql/types/Organization/updater.test.ts

```
@@ -0,0 +1,260 @@
+import type { SQL } from "drizzle-orm";
+import type { PgTableWithColumns } from "drizzle-orm/pg-core";
+import type { FastifyBaseLogger } from "fastify";
+import { beforeEach, describe, expect, it, vi } from "vitest";
+import type { organizationsTable } from "~/src/drizzle/tables/organizations";
+import type { GraphQLContext } from "~/src/graphql/context";
+import type { Organization } from "~/src/graphql/types/Organization/Organization";
+import type { User } from "~/src/graphql/types/User/User";
+import { TalawaGraphQLError } from "~/src/utilities/TalawaGraphQLError";
+import { createMockLogger } from "../../../utilities/mockLogger";
+import { usersTable } from "~/src/drizzle/tables/users"; // Import your actual users table
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)